conductor_api/src/conductor/base.rs

```diff
@@ -306,13 +337,15 @@ impl Conductor {
     /// The first time we call this, we also initialize the conductor-wide config
     /// for use with all instances
     ///
+    /// Note that the `signal_tx` parameter represents an important bifurcation of signal handling functionality.
+    /// if None, then the signal channel will be instantiated and the receive will be owned by the `Conductor`,
+    /// allowing it to automatically handle signals and push them out across the Interfaces via Broadcasters.
+    /// if it is Some, then the signal receiver is externally owned, and signals will not be sent over Interfaces.
```

Comment:
It seems important to document the Why not just the What here.  Suggestion: 
```suggestion
    /// if it is Some, then the signal receiver is externally owned, and signals will not be sent over Interfaces.  This is for the use-case of the nodejs conductor that we use for testing, which expects not to have those interfaces
```